templates/generic_show_form.html.twig

```diff
@@ -137,7 +138,7 @@
                      {% set dc_breadcrumbs %}
                         {% if item is usingtrait('Glpi\\Features\\DCBreadcrumb') %}
                            {{ call(item.getType() ~ '::renderDcBreadcrumb', [
-                              item.fields['id']
+                              item.getID
```

Comment:
You should avoid using `getID`, its returns non numeric value for new items:
```
An exception has been thrown during the rendering of a template ("DCRoom::renderDcBreadcrumb(): Argument #1 ($items_id) must be of type int, string given").
```

Reply:
(well, using `item.fields['id']` does not solve the error (I've not been able to get that foxed on my side), but anyway using getID is unfortunately not a good idea)

Reply:
Hum... OK, indeed; either I hd another method in mind, either this has changed. I guess error was from an old cache; I no longer see it